### PR TITLE
Enemy Network

### PR DESCRIPTION
**Pull request by AntonAsbrink**
Enemies are updated based on the host, they enemies will attack the closest player

### CODE REVIEW

File: StortSpelProjekt/Engine/src/Network/Network.cpp [hunk 1]
```diff
@@ -119,37 +119,23 @@ void Network::SendEnemiesPacket(std::vector<Entity*>* enemies)
     int nrOfEnemies
```
Comment (LeoWikstrom) [inline StortSpelProjekt/Engine/src/Network/Network.cpp:119]: Comments should be removed

File: StortSpelProjekt/Engine/src/Network/Network.cpp [hunk 2]
```diff
@@ -317,6 +311,36 @@ void Network::processServerData(sf::Packet* packet)
     }
 }
 
+void Network::processEnemyData(sf::Packet* packet)
+{
+    /*
```
Comment (LeoWikstrom) [inline StortSpelProjekt/Engine/src/Network/Network.cpp]: Comments should be removed

File: StortSpelProjekt/Sandbox/src/main.cpp [hunk 3]
```diff
@@ -28,6 +28,8 @@ void DemoUpdateScene(SceneManager* sm);
 
 void DefaultUpdateScene(SceneManager* sm);
 
+EnemyFactory enemyFactory;
```
Comment (LeoWikstrom) [inline StortSpelProjekt/Sandbox/src/main.cpp:30]: Should we maybe make each scene hold a pointer to their enemy factory? Maybe not in this task though...

File: StortSpelProjekt/Server/src/ClientPool.cpp [hunk 4]
```diff
@@ -135,31 +137,47 @@ void ClientPool::enemyData(int index, sf::Packet packet)
 	int nrOfEnemies
```
Comment (LeoWikstrom) [inline StortSpelProjekt/Server/src/ClientPool.cpp:137]: Remove comments

**Review by LeoWikstrom — CHANGES_REQUESTED** [review]
Looks good, works well. Just a few comments to take care of

**Review by Schtekt — CHANGES_REQUESTED** [review]
Nothing new to add from my part. Remove comments and merge in develop.

**Review by LeoWikstrom — APPROVED** [review]
🅰️

**Review by Schtekt — APPROVED** [review]
G

**Pull request merged**